src/engine/client/cl_input.cpp

```diff
@@ -195,17 +195,7 @@ float CL_KeyState( kbutton_t *key )
 
 	val = ( float ) msec / frame_msec;
 
-	if ( val < 0 )
-	{
-		val = 0;
-	}
-
-	if ( val > 1 )
-	{
-		val = 1;
-	}
-
-	return val;
+	return std::min( std::max( 0, val ), 1 );
```

Comment:
why not using Math::Clamp?

Reply:
Good idea!

Reply:
Except it's not `static_cast<float>( msec / frame_msec )` but `static_cast<float>( msec ) / frame_msec`! 😉️

```c++
int i=10;
int j=3;
printf("%f\n", (float) i/j);
printf("%f\n", static_cast<float>( i/j ));
printf("%f\n", static_cast<float>( i ) / j );
```
```
3.333333
3.000000
3.333333
```